Src/library/net/Clib/network.c

```diff
@@ -1176,8 +1176,12 @@ EIF_INTEGER c_set_sock_opt_linger(EIF_INTEGER fd, EIF_BOOLEAN flag, EIF_INTEGER
 	return (EIF_INTEGER) result;
 }
 
-EIF_INTEGER c_get_sock_recv_timeout(EIF_INTEGER fd, EIF_INTEGER level)
-	/* get socket fd SO_RCVTIMEO options */
+/********************************************************/
+/* timeout   											*/
+/********************************************************/
+
+EIF_NATURAL_64 c_get_sock_opt_timeo(EIF_INTEGER fd, EIF_INTEGER level, int optname)
```

Comment:
If this is not exported to the Eiffel code, make it private, ie prefixing with `rt_private` or `static`.
Also I'm not a big fan of the name, so why not just `c_get_timeout_with_option_name`

Reply:
this was to follow other names such as `c_set_sock_opt_linger`

Reply:
I actually do not care so much since the existing name are very confusing. At the very least it should be `timeout` and not `timeo`.